pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -343,6 +343,21 @@ func (config *V1RegistriesConf) ConvertToV2() (*V2RegistriesConf, error) {
 // anchoredDomainRegexp is an internal implementation detail of postProcess, defining the valid values of elements of UnqualifiedSearchRegistries.
 var anchoredDomainRegexp = regexp.MustCompile("^" + reference.DomainRegexp.String() + "$")
 
+// WildcardedPrefixCheck is the only place to check if a prefix contains "*."
+func WildcardedPrefixCheck(prefix string) bool {
```

Comment:
Please make this private.  We try to keep new public APIs at the bare minimum to prevent bumping the major version.